src/System.Text.Json/tests/Serialization/Value.ReadTests.cs

```diff
@@ -172,12 +173,6 @@ public static void RangeFail()
             Assert.Throws<JsonException>(() => JsonSerializer.Parse<ushort?>((ushort.MinValue - 1).ToString()));
             Assert.Throws<JsonException>(() => JsonSerializer.Parse<ushort?>((ushort.MaxValue + 1).ToString()));
 
-            // To ensure range failure, just use double's MinValue and MaxValue (instead of float.MinValue\MaxValue +-1)
-            Assert.Throws<JsonException>(() => JsonSerializer.Parse<float>(double.MinValue.ToString()));
```

Comment:
Shouldn't these continue to throw?

Reply:
Float and double do not throw - instead they will be positive\negative infinity.